ui-tests/cypress/support/pageObjects/content/versioning/VersioningPage.js

```diff
@@ -6,6 +6,12 @@ export default class VersioningPage extends AdminContent {
   searchForm      = `${htmlElements.form}[id="search"]`;
   searchDescInput = `${htmlElements.input}.form-control`;
 
+  static openPage(button) {
+    cy.intercept('http://entando7-0.apps.ent64azure.com/entando-de-app/do/jpversioning/Content/Versioning/list.action').as('contentVersioningPageLoadingGET');
```

Comment:
please, rename the _jacms_ subfolder of _controllers_ into "_action_" (it is more generic as not all this "_controllers_" are "_jacms_") and create a controller class there to perform this intercept

Reply:
I'll get right on it!